package.json

```diff
@@ -53,6 +53,9 @@
     "classnames": "^2.2.3",
     "history": "^2.0.1",
     "immutable": "^3.7.6",
+    "lodash.isfunction": "^3.0.8",
+    "lodash.set": "^4.0.0",
+    "lodash.unset": "^4.2.1",
```

Comment:
Yup, Couldn't run next-gen FRiday until i install the 3 libraries. Was expecting the same thing today . Forgot to check after getting latest. Would get rid of them